src/wp-includes/class-wp-comment-query.php

```diff
@@ -964,14 +964,12 @@ protected function get_comment_ids() {
 		$this->sql_clauses['orderby'] = $orderby;
 		$this->sql_clauses['limits']  = $limits;
 
-		$this->request = "
-			{$this->sql_clauses['select']}
-			{$this->sql_clauses['from']}
-			{$where}
-			{$this->sql_clauses['groupby']}
-			{$this->sql_clauses['orderby']}
-			{$this->sql_clauses['limits']}
-		";
+		$this->request =
+			"{$this->sql_clauses['select']}
+			 {$this->sql_clauses['from']} {$where}
```

Comment:
```suggestion
			 {$this->sql_clauses['from']}
			 {$where}
```
Having `WHERE` on a separate line seems more easily readable.